src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+@use "../utils/variable" as *;
+
+.card {
+  width: 200px;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
```

Comment:
it's redundant, just add for your picture 
`margin: 16px auto 0`
and remove align items center here